### ISSUE TITLE
Vite build fails when using string based typescript enums as parameters

### ISSUE THREAD

**Issue by samuba** [brillout/telefunc#75]
Using:
 - telefunc latest
 - vite 4.3.9 
 - svelte kit 1.20.4
 
`vite build` errors out with: `ReferenceError [Error]: t is not defined`

The enum:
```
export enum Attending {
	MAYBE = 'MAYBE',
	ATTENDING = 'ATTENDING',
	NOT_ATTENDING = 'NOT_ATTENDING',
	UNKNOWN = 'UNKNOWN',
}
```

Looking into the generated file & line we can see that after the first enum value `t.const` is being used instead of `__telefunc_t.const`
```
shield(addGuest, [__telefunc_t.optional(__telefunc_t.string), { name: __telefunc_t.string, attending: __telefunc_t.or(__telefunc_t.const("UNKNOWN"), t.const("NOT_ATTENDING"), t.const("ATTENDING"), t.const("MAYBE")) }], { __autoGenerated: true });
```

FYI: Works fine for "normal" typescript enums

**Comment by samuba** [comment]
https://stackblitz.com/edit/sveltejs-kit-template-default-lgupus?file=src%2Froutes%2F[slug]%2Fpage.telefunc.ts,src%2Froutes%2F%2Bpage.svelte

**Comment by brillout** [comment]
@louwers Any objects with https://github.com/brillout/telefunc/commit/f57cf1113b4a147765591a04501d6a3dde24ffb7?

@samuba I'll release the fix once Bart gives us the green light.

**Comment by brillout** [comment]
Fix released in `0.1.57`.

**Comment by brillout** [comment]
In case you can convince your employer: https://github.com/sponsors/brillout.
